pkg/eks/nodegroup.go

```diff
@@ -55,6 +56,26 @@ func (c *ClusterProvider) CreateOrUpdateNodeGroupAuthConfigMap(clientSet *client
 	return nil
 }
 
+//RemoveNodeGroupAuthConfigMap removes a nodegroup from the config map
+func (c *ClusterProvider) RemoveNodeGroupAuthConfigMap(clientSet *clientset.Clientset, ng *api.NodeGroup) error {
```

Comment:
Directly under `pkg`? And I'm fine with one package--it makes finding things easier.

Reply:
Or would it be in utils since kubeconfig is there?

Reply:
I orginally thought we could use `pkg/authconfigmap`, but actually `pkg/utils/authconfigmap` seem better, as you said because we have kubeconfig already, but also as some of the underlying functions already live under `pkg/utils`.

Reply:
Well, looking in `pkg/utils/kubeconfig` there's no methods/it's not using `ClusterProvider`. You're fine with those going into `pkg/utils/authconfigmap`?